Gruntfile.js

```diff
@@ -53,27 +61,60 @@ module.exports = function (grunt) {
                 dest: 'images/svg-defs.svg',
                 src: ['images/icons/**/*.svg']
             }
+        },
+        process: {
+            demo: {
+                options: {
+                    data: {
+                        svgstore: function (filename) {
+                            var contents = grunt.file.read(filename);
+                            return contents;
+                        }
+                    }
+                },
+                files: [
+                    {
+                        src: 'index.html.template',
+                        // This is committed.
+                        dest: 'index.html'
+                    }
+                ]
+            }
         }
     });
 
     // - - - T A S K S - - -
 
     grunt.registerTask('default', 'dev');
 
-    grunt.registerTask('dev', function () {
-        grunt.task.run([
-            'sass',
-            'svgstore',
-            'watch'
-        ]);
-    });
+    grunt.registerTask('dev', [
+        'sass',
+        'svgstore',
+        'watch'
+    ]);
+
+    grunt.registerTask('build', [
+        'sass',
+        'svgstore'
+    ]);
+
+    grunt.registerTask('build-demo', [
+        'build',
+        'process:demo'
+    ]);
 
-    grunt.registerTask('build', function () {
-        var tasks = [
-            'sass',
-            'svgstore'
-        ];
-        grunt.task.run(tasks);
+    grunt.registerMultiTask('process', 'Process a file wtih data', function () {
```

Comment:
There is a convenient grunt module called `grunt-template` (https://github.com/mathiasbynens/grunt-template) which will do this without needing to register a custom task